core/modules/css/CssAccess.cc

```diff
@@ -503,7 +503,8 @@ CssAccess::getPartTableParams(std::string const& dbName,
 
     std::vector<std::string> subKeys{"partitioning", "partitioning/subChunks",
         "partitioning/dirDb", "partitioning/dirTable", "partitioning/dirColName", "partitioning/latColName",
-        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName"};
+        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName",
+        "partitioning/lockInMem", "partitioning/scanSpeed"};
```

Comment:
I'm not too sure, please rethink. We have some code that blindly assumes the prefix is "partitioning", see eg "_storePacked(tableKey + "/partitioning", partMap);", or "throw KeyValueError(tableKey + "/partitioning"", that code would need to be rewritten to distinguish between the two prefixes.